spec/controllers/organizations_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'spec_helper'
+
+describe OrganizationsController do
```

Comment:
`OrganizationsController` only has the index action.
`Admin::OrganizationsController` has the other actions as well.